docs/readme.md

```diff
@@ -116,7 +116,7 @@ Alternatively, you can instruct Chrome to ignore the invalid certificate by addi
 
 Lighthouse can run against a real mobile device. You can follow the [Remote Debugging on Android (Legacy Workflow)](https://developer.chrome.com/devtools/docs/remote-debugging-legacy) up through step 3.3, but the TL;DR is install & run adb, enable USB debugging, then port forward 9222 from the device to the machine with Lighthouse.
 
-You'll likely want to use the CLI flags `--emulated-form-factor=none --throttling.cpuSlowdownMultiplier=1` to disable any additional emulation.
+You'll likely want to use the CLI flags `--no-screenEmulation --throttling.cpuSlowdownMultiplier=1` to disable any additional emulation.
```

Comment:
seems like a pattern now :) do you prefer `--no-screenEmulation` to `--no-screen-emulation`?

Reply:
with `throttlingMethod` we ended up using the camelCase version of it for the CLI in our docs, etc.   so I'm just following that.. it's nice to talk camelCase in both js config and CLI contexts.

i'd be fine with `--noScreenEmulation` but yargs doesn't do its magic in that case. :p

Reply:
haha, fair. the `--no-screenEmulation.mobile` is compelling for the camelcase version 👍 SG